src/cmd_line.rs

```diff
@@ -41,7 +41,7 @@ pub struct CmdLineSettings {
 
     /// Which window decorations to use (do note that the window might not be resizable
     /// if this is "none")
-    #[arg(long, env = "NEOVIDE_FRAME", default_value_t = Frame::default())]
+    #[arg(long, env = "NEOVIDE_FRAME", default_value_t) ]
```

Comment:
Was this intended? It's not really related to the window geometry. Though still, fine!

Reply:
Unrelated, but I noticed it was unnecessary when I was fixing the default for geometry. I can take this out if you prefer it in a separate commit.

Reply:
It's fine. I'm just not sure what clap does when you don't give a value, also the space (` `) after after the parenthesis annoys my inner eye for no reason.

Reply:
I’ll fix the space. `default_value_t` with no expression uses the Default trait for the type so you don’t have to explicitly say `= Type::default()`

Reply:
Yup, clap just seems to use the `Default` impl if not provided. I'll fix the formatting for you really quick.

Reply:
...oh whew, my GH tab didn't update so I didn't see your message in before, sorry.

Reply:
No worries!